scripts/rebase.sh

```diff
@@ -55,20 +56,20 @@ download_release() {
         >&2 echo "Warning: no pull secret found at ${PULL_SECRET_FILE}"
     fi
 
-    title "# Downloading and extracting ${release_image} tools"
-    oc adm release extract ${authentication} --tools "${release_image}"
-    echo "Content of release.txt:"
-    cat release.txt
+    title "# Fetching release info for ${release_image_amd64} (amd64)"
+    oc adm release info ${authentication} "${release_image_amd64}" -o json > release_amd64.json
+    title "# Fetching release info for ${release_image_arm64} (arm64)"
+    oc adm release info ${authentication} "${release_image_arm64}" -o json > release_arm64.json
 
-    title "# Extracing ${release_image} manifest content"
+    title "# Extracing ${release_image_amd64} manifest content"
     mkdir -p release-manifests
     pushd release-manifests >/dev/null
-    content=$(oc adm release info ${authentication} --contents "${release_image}")
+    content=$(oc adm release info ${authentication} --contents "${release_image_amd64}")
```

Comment:
Manifests are the same for any architecture, aren't they? Images will be multiarch IIUC

Reply:
Yes, they are. Or rather, they would be if both Intel and Arm release images were created together. As they are currently created independently, there could be a small delta, but that should be a rare event.

Multi-arch images are still experimental and not available for our version. Hopefully we'll be able to switch eventually.